### ISSUE TITLE
Is it possible to speed up the startup

### ISSUE THREAD

**Issue by aj3423** [equalsraf/neovim-qt#606]
There is about 1 second freeze during the startup. Normally it doesn't matter but sometimes I need to launch vim frequently. 

I tried version **0.3.8**, it's much faster, but it become slow after **0.4.x**, include the nightly build 0.5.0. 

Here's the comparation:
![nvim-qt 0 5 0](https://user-images.githubusercontent.com/4710875/68090809-3434fe00-feb3-11e9-8a9b-da37eef9a838.gif)

The above is the version 0.5.0, looks like there is a splash window, which seems not exist in 0.3.8 as below:
![nvim-qt 0 3 8](https://user-images.githubusercontent.com/4710875/68090814-53339000-feb3-11e9-93bb-55c3c3411ef9.gif)

Is it possible to skip that delay?

My environment:  Intel I7 cpu, Win10-64bit, No antivirus installed.  No vimrc used.

**Comment by equalsraf** [comment]
Is this a neovim issue or nvim-qt (asking since you mention nvim version 0.4).

If the delay is in neovim we can trackdown this we can use neovim's `--startuptime filename` to write a file with timestamps.

If the issue on the nvim-qt then we will need to look at Qt optimizations.

**Comment by aj3423** [comment]
@equalsraf The `--startuptime` shows:

```
times in msec
 clock   self+sourced   self:  sourced script
 clock   elapsed:              other lines

000.025  000.025: --- NVIM STARTING ---
011.875  011.850: locale set
012.509  000.634: inits 1
012.531  000.023: window checked
012.541  000.009: parsing arguments
012.703  000.162: expanding arguments
012.775  000.072: inits 2
016.293  003.518: init highlight
016.495  000.202: waiting for UI
181.086  164.591: done waiting for UI
181.140  000.054: initialized screen early for UI
181.221  000.081: --cmd commands
181.375  000.153: sourcing vimrc file(s)
191.508  009.930  009.930: sourcing D:\Neovim\share\nvim\runtime\filetype.vim
191.795  000.085  000.085: sourcing D:\Neovim\share\nvim\runtime\ftplugin.vim
192.026  000.068  000.068: sourcing D:\Neovim\share\nvim\runtime\indent.vim
193.073  000.571  000.571: sourcing D:\Neovim\share\nvim\runtime\syntax\syncolor.vim
193.222  000.896  000.325: sourcing D:\Neovim\share\nvim\runtime\syntax\synload.vim
193.260  001.111  000.215: sourcing D:\Neovim\share\nvim\runtime\syntax\syntax.vim
194.635  000.354  000.354: sourcing D:\Neovim\share\nvim\runtime\plugin\gzip.vim
194.765  000.045  000.045: sourcing D:\Neovim\share\nvim\runtime\plugin\health.vim
194.960  000.116  000.116: sourcing D:\Neovim\share\nvim\runtime\plugin\man.vim
196.142  000.504  000.504: sourcing D:\Neovim\share\nvim\runtime\pack\dist\opt\matchit\plugin\matchit.vim
196.199  001.152  000.648: sourcing D:\Neovim\share\nvim\runtime\plugin\matchit.vim
196.529  000.244  000.244: sourcing D:\Neovim\share\nvim\runtime\plugin\matchparen.vim
197.385  000.753  000.753: sourcing D:\Neovim\share\nvim\runtime\plugin\netrwPlugin.vim
198.230  000.229  000.229: sourcing D:\Neovim\share\nvim\runtime\autoload\remote\host.vim
198.686  000.228  000.228: sourcing D:\Neovim\share\nvim\runtime\autoload\remote\define.vim
198.793  000.965  000.508: sourcing d:\neovim\nvim-data/rplugin.vim
198.806  001.314  000.349: sourcing D:\Neovim\share\nvim\runtime\plugin\rplugin.vim
199.053  000.148  000.148: sourcing D:\Neovim\share\nvim\runtime\plugin\shada.vim
199.217  000.065  000.065: sourcing D:\Neovim\share\nvim\runtime\plugin\spellfile.vim
199.516  000.212  000.212: sourcing D:\Neovim\share\nvim\runtime\plugin\tarPlugin.vim
199.772  000.166  000.166: sourcing D:\Neovim\share\nvim\runtime\plugin\tohtml.vim
199.923  000.063  000.063: sourcing D:\Neovim\share\nvim\runtime\plugin\tutor.vim
200.218  000.207  000.207: sourcing D:\Neovim\share\nvim\runtime\plugin\zipPlugin.vim
200.870  000.429  000.429: sourcing D:\Neovim\share\nvim-qt\runtime\plugin\nvim_gui_shim.vim
200.880  003.041: loading plugins
200.970  000.091: loading packages
201.012  000.042: loading after plugins
201.074  000.061: inits 3
212.403  011.329: reading ShaDa
212.504  000.102: opening buffers
212.611  000.107: BufEnter autocommands
212.616  000.005: editing files in windows
212.842  000.226: VimEnter autocommands
212.846  000.004: UIEnter autocommands
212.848  000.002: before starting main loop
214.751  001.903: first screen update
214.755  000.004: --- NVIM STARTED ---
```

It only costs 214ms, but actually it takes more than that,  most time cost after the splash text displayed, what does nvim do after that? is it waiting for background process? It freezes the process for about 0.5 second.

**Comment by aj3423** [comment]
Seems it has been fixed on the latest NVIM v0.5.0-200-gee1199eab ,  there is still a flashing on startup, but  now it's very quick and acceptable. Thanks!
